pages/maps.tsx

```diff
@@ -29,9 +29,14 @@ import SubText from "components/common/SubText";
 import { useMyRouter } from "hooks/useMyRouter";
 import { ChangeEvent, Fragment, useEffect, useState } from "react";
 import { FiCheck, FiFilter, FiRotateCw } from "react-icons/fi";
+import { BsTable } from 'react-icons/bs';
+import { FaListUl, FaCheck } from 'react-icons/fa';
 import { shuffleArray } from "utils/arrays";
 import { stages } from "utils/lists/stages";
 import MyHead from "../components/common/MyHead";
+import { Flex } from '@chakra-ui/layout';
+import NewTable from "../components/common/NewTable";
+import {CSSVariables} from "../utils/CSSVariables";
```

Comment:
Should be caught by the pipeline but make sure to run Prettier. I think it's a command.